src/chains/ethereum/ethereum/src/forking/fork.ts

```diff
@@ -275,18 +277,11 @@ export class Fork {
       : this.blockNumber;
   }
 
-  /**
-   * If the `blockNumber` is before our `fork.blockNumber`, return a `Common`
-   * instance, applying the rules from the remote chain's `common` via its
-   * original `chainId`. If the remote chain's `chainId` is now "known", return
-   * a `Common` with our local `common`'s rules applied, but with the remote
-   * chain's `chainId`. If the block is greater than or equal to our
-   * `fork.blockNumber` return `common`.
-   * @param common -
-   * @param blockNumber -
-   */
```

Comment:
I think this comment was useful. Can we update it (maybe add in something about the timestamp stuff)

Reply:
agreed - it's a little unweildy, happy to reworder it further if you don't like it.